src/repositories/soft-crud.repository.base.ts

```diff
@@ -175,8 +175,15 @@ export abstract class SoftCrudRepository<
         [pk]: id,
       } as Condition<T>;
     }
+    if (filter.fields) {
+      filter.fields = {
+        ...filter.fields,
+        deleted: true,
+      };
+    }
```

Comment:
there should be else condition too,

Reply:
We don't need else condition, as this case will only occur only if fields property is present in the filter object.

Reply:
ok